src/connectors/snowflake/trulens/connectors/snowflake/utils/server_side_evaluation_artifacts.py

```diff
@@ -37,6 +37,7 @@
     "rich",
     "scikit-learn",
     "scipy",
+    "snowflake-ml-python",
```

Comment:
@sfc-gh-dkurokawa How do the versions get decided / picked here? 

I remember you said a while ago about how this is automatically determined.